### PR TITLE
New blog post, showcase & improvements

### PR DESCRIPTION
**Pull request by tib**
- add new blog post
- improved navigation
- new showcase

### CODE REVIEW

File: docs/news/index.html [hunk 1]
```diff
@@ -159,7 +182,7 @@ <h2><a href="https://toucansites.com/authors/andris-sipos/">Andris Sipos</a></h2
                     <img class="large rounded" src="/assets/authors/tibor-bodecs/tibor-bodecs.jpg" alt="Tibor Bödecs">
                 </a>
                 <h2><a href="https://toucansites.com/authors/tibor-bodecs/">Tibor Bödecs</a></h2>
-                <p>0 articles</p>
+                <p>1 articles</p>
```
Comment (viaszkadi) [inline docs/news/index.html:185]: Some of the pages have this issue. I believe it’s not very important for now, but I wanted to mention it.

File: docs/toucan-is-now-ready-for-apha-testing/index.html [hunk 2]
```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en-US">
+<head>
+    <meta charset="utf-8">
+    
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+    <link rel="canonical" href="https://toucansites.com/toucan-is-now-ready-for-apha-testing/">
```
Comment (viaszkadi) [inline docs/toucan-is-now-ready-for-apha-testing/index.html:8]: `apha` => `alpha` Look for more occurrences, including in the file name.

File: docs/toucan-is-now-ready-for-apha-testing/index.html [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+    
+    <link rel="next" href="[&quot;expiration&quot;: false, &quot;js&quot;: [], &quot;permalink&quot;: &quot;https://toucansites.com/introducing-toucan-a-new-markdown-based-static-site-generator/&quot;, &quot;description&quot;: &quot;We\&#39;re thrilled to announce the development of Toucan, our new lightweight static site generator built using Swift.&quot;, &quot;title&quot;: &quot;Introducing Toucan: a new Markdown-based static site senerator&quot;, &quot;css&quot;: [], &quot;imageUrl&quot;: false, &quot;hreflang&quot;: [], &quot;noindex&quot;: false, &quot;canonical&quot;: &quot;https://toucansites.com/introducing-toucan-a-new-markdown-based-static-site-generator/&quot;, &quot;slug&quot;: &quot;introducing-toucan-a-new-markdown-based-static-site-generator&quot;, &quot;publication&quot;: ToucanSDK.PageBundle.Context.DateValue(html: &quot;2024.08.12.&quot;, rss: &quot;Mon, 12 Aug 2024 12:00:00 +0000&quot;, sitemap: &quot;2024-08-12&quot;), &quot;lastModification&quot;: ToucanSDK.PageBundle.Context.DateValue(html: &quot;2024.08.13.&quot;, rss: &quot;Tue, 13 Aug 2024 11:56:21 +0000&quot;, sitemap: &quot;2024-08-13&quot;)]">
```
Comment (viaszkadi) [inline docs/toucan-is-now-ready-for-apha-testing/index.html:10]: `senerator` => `generator`

File: docs/toucan-is-now-ready-for-apha-testing/index.html [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+        <br>
+        <div class="grid grid-221">
+            <div class="post card">
+                
+            
+                <div class="meta">
+                    <time datetime="2024.08.12.">2024.08.12.</time>
+                    
+                </div>
+                <h2 class="title"><a href="https://toucansites.com/introducing-toucan-a-new-markdown-based-static-site-generator/" target="">Introducing Toucan: a new Markdown-based static site senerator</a></h2>
```
Comment (viaszkadi) [inline docs/toucan-is-now-ready-for-apha-testing/index.html:179]: `senerator` => `generator`

File: src/content/blog/articles/toucan-is-now-ready-for-apha-testing/index.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+That’s all about the toucan command. Now, let’s focus on building a simple website.
+
+## Getting started with Toucan-based websites
+
+The first thing you need is a source folder where you can store all the themes, assets, and content for your website.
+
+Feel free to grab one of our themes to get started. We also have a minimal starter project available under the [toucansites](https://github.com/toucansites) GitHub organization.
+
+Make sure to place the theme in the `themes/default`folder. Alternatively, you can set the `theme.use` value to use a custom folder.
```
Comment (viaszkadi) [inline src/content/blog/articles/toucan-is-now-ready-for-apha-testing/index.md:64]: `themes/default`folder => `themes/default` folder

File: src/content/blog/articles/toucan-is-now-ready-for-apha-testing/index.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+│           └── index.md
+├── index.md
+└── my-page
+    └── index.md
+```
+
+The following pages will be available:
+
+- `/ `
+- `404 `
```
Comment (viaszkadi) [inline src/content/blog/articles/toucan-is-now-ready-for-apha-testing/index.md:122]: Do we need spacing here and above?

**Review by viaszkadi — CHANGES_REQUESTED** [review]

**Review by mzpx — APPROVED** [review]

**Review by viaszkadi — APPROVED** [review]

**Review by mzpx — APPROVED** [review]

**Pull request merged**